src/pages/basic_data/institution_admin/AddEmployeePage.tsx

```diff
@@ -113,35 +113,28 @@ export const AddEmployeePage = () => {
             <Box
               sx={{
                 display: 'grid',
-                gridTemplateColumns: { xs: '1fr', md: '1fr auto 1fr auto 1fr' },
+                gridTemplateColumns: { xs: '1fr', md: '3fr auto 2fr' },
                 gap: '1rem',
                 mt: '2rem',
               }}>
               <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
                 <FindPersonPanel />
               </Box>
               <Divider orientation="vertical" />
-              <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+              <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', justifyContent: 'right' }}>
```

Comment:
Har `right` noen effekt da? Må man ikke bruke `end`? 🤔 
```suggestion
              <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
```

Reply:
Den hang igjen fra når jeg prøvde noe flex greier på parenten, men var overflødig nå. Men mener at jeg måtte bruke `right` når det var `flexDirection: 'column'`.

Reply:
lett å glemme å oppdatere følger i children når man endrer noe i parent ja!